### DISCUSSION TITLE
Warning: No primary keys selected

### DISCUSSION THREAD

**Discussion by proko** [supabase/supabase#3459] [Questions]
Hello! Anyone else having this warning in the table configuration screen? Even if I set a primary key and save, I get the confirmation that it got saved but when coming back, warning is back and no primary keys set. Happens to all tables. 

It's apparently not a bug since I can't find anyone else complaining about this so I must be missing something. 

<img width="665" alt="Screenshot 2021-10-08 at 22 17 02" src="https://user-images.githubusercontent.com/3257883/136620227-a345b3b2-4793-476e-8b48-329d6cb4dd06.png">

**Comment by trinhdh** [comment]
I do. Even when i select primary for`id` and save it. The warning still showing after the page reload. However, when i using PgAdmin to connect to my Supabase database using connection pool. Its showing correctly ( `id` is PRIMARY KEY). 
So probably, there is some bug in Supabase dashboard.

**Reply by proko** [reply]
Good to know, thanks @trinhdh! Last time I tried to use the tables I was getting some errors because of that (I think) and this is how I got to see the bug. I will give it another go. Might have been failing for a completely unrelated reason!
